website/addons/box/templates/log_templates.mako

```diff
@@ -1,6 +1,6 @@
 <script type="text/html" id="box_file_added">
 added file
-<a class="overflow log-file-link" data-bind="click: NodeActions.addonFileRedirect">{{ params.path }}</a> to
+<a class="overflow log-file-link" data-bind="click: NodeActions.addonFileRedirect">{{ params.fullPath }}</a> to
```

Comment:
It will always and only be available for upload/updates, it gets computed and inserted into the [metadata](https://github.com/mfraezz/waterbutler/blob/hotfix/box-full-pathname/waterbutler/providers/box/provider.py#L71) that waterbutler returns on that call.